src/client/api/pythonApi.ts

```diff
@@ -425,7 +406,6 @@ export class InterpreterService implements IInterpreterService {
                     api.onDidChangeInterpreter(
                         () => {
                             this.didChangeInterpreter.fire();
-                            this.workspaceCachedActiveInterpreter.clear();
```

Comment:
At one point Python extension was not caching this, and calling this Python extension API was pretty expensive, are they caching this now?
If not it could adversely impact our code (extension).

Reply:
Not sure, trying to figure out why the active interpeter seems to not match the ```python.pythonPath``` in some of the tests. I can put this back later if I can figure out why.